File: 002/Problem2.go
package main

import (
	"fmt"
	"time"
)

func problem002() {
	var a, b, sum int = 1, 2, 2
	var c int

	for c < 4_000_000 {
		c = a + b
		if c%2 == 0 {
			sum += c
		}
		a = b
		b = c
	}

	fmt.Println(sum)
}

func main() {
	start := time.Now()
	problem002()
	elapsed := time.Since(start)
	fmt.Printf("Function took %s", elapsed)
}

File: 002/Problem2.py
import time


def problem001():
    a = 1
    b = 2
    su = 2
    c = 0

    while c < 4_000_000:
        c = a + b
        if c % 2 == 0:
            su += c
        a = b
        b = c
    print(su)


start = time.time()
for i in range(1000):
    problem001()
end = time.time()
print("Function took " + str(end - start))
